src/main/java/stepsdesign/CdfHubSteps.java

```diff
@@ -0,0 +1,126 @@
+/*
+ * Copyright © 2023 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package stepsdesign;
+
+import io.cdap.e2e.pages.actions.CdfHubActions;
+import io.cdap.e2e.pages.locators.CdfHubLocators;
+import io.cdap.e2e.utils.CdfHelper;
+import io.cdap.e2e.utils.SeleniumHelper;
+import io.cucumber.java.en.Then;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * CDF hub related step design.
+ */
+public class CdfHubSteps implements CdfHelper {
+  private static final Logger logger = LoggerFactory.getLogger(CdfHubSteps.class);
+  static {
+    SeleniumHelper.getPropertiesLocators(CdfHubLocators.class);
+  }
+
+  @Then("Click on Hub Menu")
+  public static void openHub() {
+    CdfHubActions.clickOnHub();
+  }
```

Comment:
Add space post each actions is declared . line number 39,43,47 and so on ..

Reply:
done